.env

```diff
@@ -58,7 +58,7 @@ SPDK_CENTOS_REPO_VER="9.0-21.el9"
 
 # Ceph Cluster
 CEPH_CLUSTER_VERSION="${CEPH_VERSION}"
-# CEPH_CLUSTER_CEPH_REPO_BASEURL=
+CEPH_CLUSTER_CEPH_REPO_BASEURL=https://3.chacra.ceph.com/r/ceph/ceph-nvmeof-mon/c1ced49d133c2bcd1fa9311a35688bf641b93c76/centos/9/flavors/default
```

Comment:
@gbregman @epuertat  so cool 🎉

please share your thought about the following approach:
1. define in `.env` 
```shell
CEPH_BRANCH=ceph-nvmeof-mon
CEPH_SHA=c1ced49d133c2bcd1fa9311a35688bf641b93c76
```
2. in Makefile, under [this line](https://github.com/ceph/ceph-nvmeof/blob/ac5a7ad3b28e5312fd9446e01b3d45990559ea88/Makefile#L35)
```Makefile
build: export CEPH_CLUSTER_CEPH_REPO_BASEURL != curl -s https://shaman.ceph.com/api/repos/ceph/$CEPH_BRANCH/$CEPH_SHA/centos/9/ | jq -r '.[0].url'
```

Would that work?

Reply:
@baum this seems to work for this commit. I'm not sure it would work in the general case but I guess as this is a temporary feature we can use that. I